src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -67,8 +88,38 @@ public void ConfigureServices(IServiceCollection services)
 #endif
 #if CertificateAuth
                 options.Filters.Add(new CertificateAuthenticationFilter());
+#endif
+#if JwtAuth
+                AuthorizationPolicy policy = 
+                    new AuthorizationPolicyBuilder()
+                        .RequireRole("Admin")
+                        .RequireAuthenticatedUser()
+                        .Build();
+
+                options.Filters.Add(new AuthorizeFilter(policy));
 #endif
             });
+#if JwtAuth
+            var key = Encoding.ASCII.GetBytes(Configuration["Jwt:Secret"]);
+            services.AddAuthentication(x =>
+                    {
+                        x.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                        x.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+                    })
+                    .AddJwtBearer(x =>
+                    {
+                        x.SaveToken = true;
+                        x.TokenValidationParameters = new TokenValidationParameters
+                        {
+                            ValidateIssuerSigningKey = true,
+                            IssuerSigningKey = new SymmetricSecurityKey(key),
+                            ValidateIssuer = true,
+                            ValidIssuer = "entity that generates the token",
```

Comment:
I think this one should be configurable as well?

Reply:
Agreed